pkg/controllers/fixtures/images.json

```diff
@@ -1,4 +1,5 @@
 {
+    "cloudControllerManagerOperator": "quay.io/openshift/origin-cluster-cloud-controller-manager-operator",
```

Comment:
Should we make sure the pattern matches the others? Use the registry.ci tag instead?